Adding a "global" not_intent parameter for slot mappings (#7931)
RasaHQ/rasa: changelog/261.removal.md

> In forms, the keyword `required_slots` should always precede the definition of slot mappings and the lack of it will be deprecated.

Reviewer: Can we please link to the docs for this?
```suggestion
In forms, the keyword `required_slots` should always precede the definition of slot mappings and the lack of it is deprecated.
```

In forms, the keyword `required_slots` should always precede the definition of slot mappings and the lack of it is deprecated.

> Please see https://rasa.com/docs/rasa/forms for more information.

Reviewer: ```suggestion
Please see the [migration guide](migration-guide.mdx) for more information.
```
